src/webapp/src/auth/AuthenticationOptions.tsx

```diff
@@ -1,7 +1,68 @@
+import { Button } from '@small-ads/ui';
 import React from 'react';
+import styled from 'styled-components';
+import { FacebookSquare } from '@styled-icons/boxicons-logos';
+
+const LoginScreen = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: calc(100vh - 64px);
+`;
+
+const LoginMain = styled.div`
+  align-self: center;
+  flex: 1 1;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+`;
+
+const LoginSign = styled.div`
+  font-size: 3rem;
+  letter-spacing: -0.066875rem;
+  font-weight: 700;
+`;
+
+const LoginForm = styled.div`
+  width: 100%;
+  text-align: center;
+  max-width: 320px;
+  align-self: center;
+`;
+
+const GeistSpacer = styled.span`
```

Comment:
What `Geist` means for us?

Reply:
I copied the name from Vercel, I understood that `Geist` was like a `ghost` component to make space. Should we name it `GhostSpacer`?

Reply:
`GhostSpacer` sounds cool yeah